plugins/UserCountryMap/javascripts/vendor/kartograph.min.js

```diff
@@ -15,7 +15,9 @@
  *  You should have received a copy of the GNU Lesser General Public
  *  License along with this library. If not, see <http://www.gnu.org/licenses/>.
  *
+ * NOTE, manual modification made:
+ * - removed Function.bind implementation, does not work properly w/ angular 1.6
```

Comment:
Is this file supposed to be changed?

Reply:
Should have been changed in the upgrade to angular 1.6, not sure why it's changed here. I'll take a look.

Reply:
Oh I didn't rebase the PR.